<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>383</width>
    <height>700</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>JetBrains Mono</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>BandMap</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="leftMargin">
     <number>6</number>
    </property>
    <property name="topMargin">
     <number>6</number>
    </property>
    <property name="rightMargin">
     <number>6</number>
    </property>
    <property name="bottomMargin">
     <number>6</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Callsign</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="callsignField">
        <property name="accessibleName">
         <string>callsign</string>
        </property>
        <property name="accessibleDescription">
         <string>callsign used to sign in to d x cluster.</string>
        </property>
        <property name="styleSheet">
         <string notr="true">text-transform: uppercase;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="connectButton">
        <property name="accessibleName">
         <string>connect</string>
        </property>
        <property name="accessibleDescription">
         <string>connect to d x cluster.</string>
        </property>
        <property name="text">
         <string>Connect</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="font">
       <font>
        <family>JetBrains Mono</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>367</width>
         <height>546</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>JetBrains Mono</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QGraphicsView" name="graphicsView">
          <property name="font">
           <font>
            <family>JetBrains Mono</family>
            <pointsize>6</pointsize>
           </font>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="accessibleName">
           <string>spots</string>
          </property>
          <property name="accessibleDescription">
           <string>spots from cluster.</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>20</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="clearButton">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="accessibleName">
         <string>clear all</string>
        </property>
        <property name="accessibleDescription">
         <string>clear spots</string>
        </property>
        <property name="text">
         <string>Clear All</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="zoominButton">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="accessibleName">
         <string>zoom in</string>
        </property>
        <property name="accessibleDescription">
         <string>zoom in</string>
        </property>
        <property name="text">
         <string>Zoom In</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="zoomoutButton">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="accessibleName">
         <string>zoom out</string>
        </property>
        <property name="accessibleDescription">
         <string>zoom out</string>
        </property>
        <property name="text">
         <string>Zoom Out</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="clear_spot_olderLabel">
        <property name="font">
         <font>
          <family>JetBrains Mono</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Clear older than </string>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="clear_spot_olderSpinBox">
        <property name="accessibleName">
         <string>clear spots older than</string>
        </property>
        <property name="accessibleDescription">
         <string>removes spots older than x amount of minutes ago.</string>
        </property>
        <property name="specialValueText">
         <string>Never</string>
        </property>
        <property name="suffix">
         <string> min(s)</string>
        </property>
        <property name="maximum">
         <number>9999</number>
        </property>
        <property name="value">
         <number>1</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
